test/skimNtuple2018.cpp

```diff
@@ -5103,6 +5678,292 @@ int main (int argc, char** argv)
 
   } // End new BDT
 
+/*
```

Comment:
I know I had told you to comment it out for the moment, but can you uncomment this DNN part?